### ISSUE TITLE
bootstrap.nlsfit runs something in parallel even though parallel = FALSE has been passed

### ISSUE THREAD

**Issue by kostrzewa** [HISKP-LQCD/hadron#188]

**Comment by j-ostmeyer** [comment]
That's a total mystery to me. I just searched through the complete code and there is no place mentioning `parallel` or doing something with `mc` like `mclapply` where it should not. As far as I know, none of the used routines runs in parallel by default.

**Comment by j-ostmeyer** [comment]
Is it happening with both `optim` and `nls.lm`?

**Comment by kostrzewa** [comment]
Yes, I will need to try it out in a more isolated situation, however, to be 100% sure.
